package.json

```diff
@@ -21,6 +21,7 @@
     "@observablehq/plot": "^0.2.0",
     "d3-array": "^3.0.2",
     "d3-format": "^3.0.1",
+    "d3-time": "^3.0.0",
```

Comment:
ahah whoops, I bet this only worked because observable includes all of d3

Reply:
> ahah whoops, I bet this only worked because observable includes all of d3

Yeah that sounds about right